webapp/first_snap/content/java/package.yaml

```diff
@@ -0,0 +1,69 @@
+name: freeplane-{name}
+download: git clone https://github.com/snapcraft-docs/freeplane
+createDir: |
+  cd freeplane
+  $EDITOR snapcraft.yaml
+metadata: |
+  name: freeplane
+  version: git
+  summary: Application for Mind Mapping, Knowledge and Project Management
+  description: |
+    Freeplane is a free and open source software application that supports thinking, sharing information and getting things done at work, in school and at home. The core of the software is tools for mind mapping (also known as concept mapping or information mapping) and using mapped information.
+explain:
+  metadata:
+    - text: The name must be unique in the Snap Store. Valid snap names consist of lower-case alphanumeric characters and hyphens. They cannot be all numbers. They also cannot start or end with a hyphen.
+    - code: |
+        name: freeplane
+    - warning: |
+        <span class="p-notification__status">Snap names are globally unique.</span> You should change any instance of <code>freeplane</code> to <code>freeplane-{name}</code> (where <code>{name}</code> is your name) before continuing to avoid conflicting with the official freeplane snap.
+    - text: Versions carry no semantic meaning in snaps.
+    - code: |
+        version: '1.6.10' 
+    - text: The summary can not exceed 79 characters. You can use a pipe ‘|’ in the description key to declare a multi-line description.
+    - code: |
+        description: |
+  security:
+    - text: The next section describes the level of confinement applied to your app.
+    - code: |
+        confinement: devmode
+    - text: Snaps are containerised to ensure more predictable application behaviour and greater security.
+    - text: It’s best to start a snap with the confinement in warning mode, rather than strictly applied. This is indicated through the <code>devmode</code> keyword.
+    - text: Once an app is working well in devmode, you can review confinement violations, add appropriate interfaces, and switch to strict confinement.
+  parts:
```

Comment:
Congratulations, you found a bug:
<img width="230" alt="screenshot 2019-02-07 at 18 30 01" src="https://user-images.githubusercontent.com/1523179/52434091-75f06e00-2b06-11e9-97d3-b4b0451bad89.png">

We should fix as part of this PR.